code/game/objects/machinery/overwatch.dm

```diff
@@ -199,7 +194,9 @@ GLOBAL_LIST_EMPTY(active_cas_targets)
 	name = "\improper Whiskey Overwatch Console"
 
 /obj/machinery/computer/camera_advanced/overwatch/CreateEye()
-	. = ..()
```

Comment:
Should this have been removed?

Reply:
ya because the parent makes a `eyeobj` too